language/sygus.tex

```diff
@@ -1816,14 +1817,16 @@ \subsubsection{Weights}
 
```

Comment:
In the `cases` block above, we have `Wk` in the first case.
Should it be `W k` (with a space between `W` and `k`)?

Reply:
Yes, it seems the formatting was off, I'll fix.